tms_objects2.erb

```diff
@@ -561,6 +563,7 @@ objects_main_query = tmsclient.execute("SELECT * FROM [Coboat_ObjectsMainQuery]
 #9 - ExhTextEntries |t| - displays eventDescriptionSet
 #308 - attributionQualifierActor - PubCreatorSet |pc| - Prefix - Published by
 # No DisplayBio in lido:TMS
+# 4368 - multiple results for PubConLifeRole within roleActor
```

Comment:
I don't see a publication event PubConLifeRole actor in 4368

Reply:
Also, for consistency add the unless nil logic for attributionQualifierActor in production event